vue/sbc-common-components/src/components/BaseAddress.vue

```diff
@@ -255,25 +264,30 @@ export default class BaseAddress extends Mixins(ValidationMixin, CountriesProvin
   ]
 
   /**
-   * Getter for Vuetify rules object. Used to display any validation errors/styling.
-   * @remark As a getter, this is initialized between created() and mounted().
-   * @returns The Vuetify validation rules object.
+   * The Vuetify rules object. Used to display any validation errors/styling.
+   * NB: As a getter, this is initialized between created() and mounted().
+   * @returns the Vuetify validation rules object
    */
   private get rules (): { [attr: string]: Array<Function> } {
     return this.createVuetifyRulesObject('addressLocal')
   }
 
-  /**
-   * Emits an update message for the {@link address} property, so that the caller can ".sync" with it.
-   */
+  /** Emits an update message for the address prop, so that the caller can ".sync" with it. */
   @Emit('update:address')
-  private emitAddress (val: object): void { }
+  private emitAddress (address: object): void { }
+
+  /** Emits the validity of the address entered by the user. */
+  @Emit('valid')
+  private emitValid (valid: boolean): void { }
 
   /**
-   * Emits the Vuelidate state of the address entered by the user.
+   * Watches changes to the Schema object, so that if the parent changes the data, then
+   * the working copy of it is updated.
    */
-  @Emit('valid')
-  private emitValid (val: boolean): void { }
+  @Watch('schema', { deep: true, immediate: true })
+  private onSchemaChanged (): void {
+    this.schemaLocal = { ...this.schema }
```

Comment:
I made this a watcher because the previous code was giving me the "using a property before it's declared" error. So it's set immediately which does the same thing but without the error.